README.md

```diff
@@ -22,3 +22,14 @@ Outputs: `./index.js` and `lib/*.js`
 1. `npm login`
 2. `npm run publish`
 3. Answer questions from np
+
+## UI Component Directory Structure
+
```

Comment:
This is the directory structure I am proposing with this PR. It make it easy for us to bundle individual UI components, and eliminates some redundancy in the paths